climada/util/config.py

```diff
@@ -50,6 +50,21 @@ class Config():
     Like this it's possible to refer to e.g. test files in the climada sources.
     """
 
+    def __getattribute__(self, __name):
+        # the only reason for overriding __getattribute__: better error messages in case a
+        # configuration value is asked for but never defined in any of the possible config files
+        try:
+            return super().__getattribute__(__name)
+        except AttributeError:
+            conf_files = [Path(_find_in_parents(conf_dir, CONFIG_NAME))
+                          if _find_in_parents(conf_dir, CONFIG_NAME)
+                          else conf_dir / CONFIG_NAME
```

Comment:
```suggestion
                          else Path(conf_dir / CONFIG_NAME)
```
Would this not be more consistent?

Reply:
:shrug: `conf_dir` is a path though and even with the proposed change it must be one.